packages/telemetry-e2e/values.yaml

```diff
@@ -0,0 +1,182 @@
+global:
+  domain: .change.me
+  cluster: minikube
+
+kafka:
+  # password for the Kafka user generated to give Ditto access to the event topic for "eclipse-iot"
+  password: changeme
+
+drogueApplication:
+  name: eclipse
+
+demoDevice:
+  id: device-1
+  password: device12
+
+drogueCloud:
+
+  defaults:
+    ingress:
+      className: nginx
+
+  endpoints:
+    coap:
+      insecure: true
+      ingress:
+        port: 30003
+    mqtt:
+      insecure: true
+      ingress:
+        port: 30001
+    http:
+      insecure: true
+      ingress:
+        port: 30443
+
+  integrations:
+    mqtt:
+      insecure: true
+      ingress:
+        port: 30002
+    websocket:
+      insecure: true
+      ingress:
+        port: 30004
+
+  services:
+    console:
+      ingress: {}
+    api:
+      ingress: {}
+    sso:
+      ingress:
+        annotations:
+          nginx.ingress.kubernetes.io/backend-protocol: HTTPS
+          nginx.ingress.kubernetes.io/server-snippet: |2-
+                              location ~* "^/auth/realms/master/metrics" {
+                                  return 301 /auth/realms/master;
+                              }
+
+  keycloak:
+    resources:
+      requests:
+        memory: 128Mi
+        cpu: "100m"
+      limits:
+        memory: 1Gi
+        cpu: "1"
+
+  kafka:
+    kafka:
+      resources:
+        requests:
+          memory: 128Mi
+          cpu: "100m"
+        limits:
+          memory: 1Gi
+          cpu: "1"
+    zookeeper:
+      resources:
+        requests:
+          memory: 128Mi
+          cpu: "100m"
+        limits:
+          memory: 512Mi
+          cpu: "500m"
+    topicOperator:
+      resources:
+        requests:
+          memory: 128Mi
+          cpu: "100m"
+        limits:
+          memory: 256Mi
+          cpu: "250m"
+    userOperator:
+      resources:
+        requests:
+          memory: 128Mi
+          cpu: "100m"
+        limits:
+          memory: 256Mi
+          cpu: "250m"
+
+ditto:
+  fullnameOverride: ditto
+  dittoTag: 2.1.0-M3
```

Comment:
this can be dropped now in favor of using the latest 2.1 Ditto chart with final Ditto 2.1.0 in it